test/test-get-passwd.c

```diff
@@ -64,19 +64,23 @@ TEST_IMPL(get_passwd) {
   ASSERT(pwd.gid >= 0);
 #endif
 
+  ASSERT(pwd.gecos == NULL || strlen(pwd.gecos) >= 0);
```

Comment:
This warning because `strlen` cannot return a negative value:
```
In file included from test/test-get-passwd.c:23:0:
test/test-get-passwd.c: In function ‘run_test_get_passwd’:
test/test-get-passwd.c:67:49: warning: comparison of unsigned expression >= 0 is always true [-Wtype-limits]
   ASSERT(pwd.gecos == NULL || strlen(pwd.gecos) >= 0);
                                                 ^
test/task.h:88:9: note: in definition of macro ‘ASSERT’
   if (!(expr)) {
```

Reply:
Yea, I guess I should drop the assertion completely? Originally, I checked for `NULL` and `strlen() > 0`, but the empty string is valid here.